addons/web/static/src/js/views/form/form_renderer.js

```diff
@@ -1019,17 +1019,10 @@ var FormRenderer = BasicRenderer.extend({
      * @override method from BasicRenderer
      * @returns {Promise}
      */
-    _renderView: function () {
+    _renderView: async function () {
         var self = this;
-
-        // render the form and evaluate the modifiers
-        var defs = [];
-        this.defs = defs;
-        this.inactiveNotebooks = [];
-        var $form = this._renderNode(this.arch).addClass(this.className);
-        delete this.defs;
-
-        return Promise.all(defs).then(function () {
+        const $form = await this.__renderView();
+        return Promise.resolve().then(function () {
```

Comment:
```suggestion
        return this.__renderView().then(function ($form) {
```

Reply:
`$form` is needed in the `guardedCatch` block. I could make it a dynamic var but the line count would be the same.

Reply:
alright, read too fast